test/test_utils.py

```diff
@@ -223,7 +226,8 @@ def test_draw_segmentation_masks(colors, alpha):
     assert_equal(img[:, ~masked_pixels], out[:, ~masked_pixels])
 
     if colors is None:
-        colors = utils._generate_color_palette(num_masks)
+        palette = torch.tensor([2**25 - 1, 2**15 - 1, 2**21 - 1])
+        colors = [tuple((i * palette) % 255) for i in range(num_masks)]
```

Comment:
Hmmm, I was not aware of this usage. Maybe we should have the `_generate_color_palette` function after all? I lean towards putting it back, but no strong opinion.

Reply:
I agree, will put it back